python/src/lib/s3_helper.py

```diff
@@ -56,3 +56,37 @@ def upload_generated_file_to_s3(
         raise
 
     logger.info("successfully uploaded file to s3")
+
+
+def get_presigned_url(
+        s3_client: S3Client,
+        bucket: str,
+        key: str,
+        expiration_time: int = 60 * 60,  #  1 hour
+):
+    logger = get_logger()
+    try:
+        response = s3_client.head_object(
+            bucket = bucket,
+            key = key,
+        )
+    except ClientError as e:
+        if logger:
```

Comment:
This `if logger:` check shouldn't be necessary. Was this added due to issues with testing the functionality?

Reply:
Yes, I'll remove it. I don't think it's necessary anymore.